Use single equals in makefile

tcharding opened the pull request:
Currently we use double equals ('==') within a shell test statement that uses single square brackets.  The standard comparison operator is '=' not '==' for single square brackets.   Although '==' works in `bash` it does
not work in all shells (e.g. `zsh`).

Use standard comparison operation '='

Signed-off-by: Tobin C. Harding <me@tobin.cc>

Waldz commented:
@tcharding, would You mind to setup GGP and sign your commit 
https://help.github.com/articles/signing-commits-using-gpg/

tcharding commented:
I've been on a plane all night without sleeping so excuse if this is a braindead comment but I do not know why the merge of master into this branch?  FTR I added my gpg key to github and configured this repo to always sign.  Rebased and signed the original commit and force pushed.  I have not see 'dismissed stale reviews' before?

tcharding commented:
Closing this because the Makefile has changed since the PR was open.  FTR this issue exist now for the code in `make dep`.

thanks
